winsup/utils/kill.cc

```diff
@@ -162,7 +163,10 @@ forcekill (int pid, int sig, int wait)
   if (pid > 0)
     p = (external_pinfo *) cygwin_internal (CW_GETPINFO_FULL, pid);
   DWORD dwpid = p ? p->dwProcessId : (DWORD) pid;
-  HANDLE h = OpenProcess (PROCESS_TERMINATE, FALSE, (DWORD) dwpid);
+  HANDLE h = OpenProcess (PROCESS_CREATE_THREAD | PROCESS_QUERY_INFORMATION |
+			  PROCESS_VM_OPERATION | PROCESS_VM_WRITE |
+			  PROCESS_VM_READ | PROCESS_TERMINATE,
+			  FALSE, (DWORD) dwpid);
   if (!h)
```

Comment:
The new code asks for more permissions than the old code in the OpenProcess() call.  If this call fails, could it be because we don't have sufficient access?  Should we fall back here and retry the OpenProcess() with the original parameters and then use the original kill code?